themes/bootstrap3/templates/record/view.phtml

```diff
@@ -7,6 +7,11 @@
   // Set up standard record scripts:
   $this->headScript()->appendFile('record.js');
   $this->headScript()->appendFile('check_save_statuses.js');
+
+  // Add carousel library for Similar Items
+  $this->headScript()->appendFile('vendor/splide.min.js');
```

Comment:
If we want to avoid putting this in such a global place, you could move these to RecordTab/similaritemscarousel.phtml and adjust the VuFind\RecordTab\SimilarItemsCarousel class to disallow AJAX loading. Or maybe there's a way to dynamically load the content as part of the template even if it is fetched by AJAX (I'm pretty sure you can get away with that for Javascript, but I'm less sure about the CSS).